pyproject.toml

```diff
@@ -51,6 +51,7 @@ poetry-core = "1.4.0"
 poetry-plugin-export = "^1.2.0"
 "backports.cached-property" = { version = "^1.0.2", python = "<3.8" }
 cachecontrol = { version = "^0.12.9", extras = ["filecache"] }
+lockfile = "^0.12.2"
```

Comment:
this list is in alphabetical order (aside from `poetry-*`)

Reply:
Fixed! Thank you, didn't see that it was sorted.